packages/flutter/lib/src/services/text_input.dart

```
@@ -1547,7 +1573,7 @@ class TextInput {
   @visibleForTesting
   static void setChannel(MethodChannel newChannel) {
     assert(() {
-      _instance._channel = newChannel..setMethodCallHandler(_instance._handleTextInputInvocation);
+      _instance._channel = newChannel..setMethodCallHandler(_instance._loudlyhandleTextInputInvocation);
```

Are all of these method call handlers swallowing errors?  Maybe we should make them all "loud" long term...

---

Possibly for other official channels. But maybe not implicitly for `setMethodCallHandler`.

---

I opened an issue for discussion: https://github.com/flutter/flutter/issues/105566